README.md

```diff
@@ -6,60 +6,75 @@
 
 Reference implementation for the Connected Diagnostics API (http://dxapi.org/)
 
-## Getting Started
+## CDX Core Server. Getting Started
 
-### CDX Core Server
-
-To start developing:
+### Development environment (with Docker)
 
 1. Clone the repo.
 
-2. Install dependencies:
-	* `bundle install`.
-	* PhantomJS 1.9.8 for [Poltergeist](https://github.com/teampoltergeist/poltergeist) (development and test only)
-		* Install it in mac with: `brew install phantomjs`
-	* ImageMagick for [Paperclip](https://github.com/thoughtbot/paperclip#image-processor)
-		* Install it in mac with: `brew install imagemagick`
-	* [Redis](http://redis.io/download) is [used](https://github.com/mperham/sidekiq/wiki/Using-Redis) by [sidekiq](http://sidekiq.org/). CDX uses sidekiq as [ActiveJob](http://guides.rubyonrails.org/active_job_basics.html#backends) backend
-		* Install it in mac with: `brew install redis`
-		* you can start it with `redis-server --daemonize yes`
-	* [Elasticsearch](https://www.elastic.co/) is used as the main index for test results.
-		* We support elasticsearch versions < 2.x
-		* Install it in mac with: `brew install elasticsearch17`
+2. Install Docker.
 
-3. Setup development database: `bundle exec rake db:setup`
+3. Install `gems`:
+	```
+	$ docker-compose build
+	$ docker-compose run --rm web gem install bundler -v `<2`
+	$ docker-compose run --rm web bundle install
```

Comment:
Do we really need to `gem install bundler`?

Reply:
yep, you're right! We don't need it!